arroyo/backends/kafka/commit.py

```diff
@@ -6,29 +7,31 @@
 from arroyo.types import Partition, Topic
 from arroyo.utils.codecs import Codec
 
+# Kept in decode method for backward compatibility. Will be
+# remove in a future release of Arroyo
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 
 
 class CommitCodec(Codec[KafkaPayload, Commit]):
     def encode(self, value: Commit) -> KafkaPayload:
         assert value.orig_message_ts is not None
 
+        payload = json.dumps(
+            {
+                "offset": value.offset,
+                "orig_message_ts": datetime.timestamp(value.orig_message_ts),
```

Comment:
Is this for the alerts e2e SLO ?

Reply:
Not really, this is the same information that was already there, just in the body not the header now so we don't need to do the custom decoding dance